notices.json

```diff
@@ -110,5 +110,23 @@
         "actionParam": "https://mattermost.com/blog/collapsed-reply-threads-beta/"
       }
     }
+  },
+  {
+    "id": "v6.0_user_introduction",
+    "conditions": {
+      "audience": "all",
+      "clientType": "all",
+      "serverVersion": [">=6.0"],
```

Comment:
@ogi-m what happens here for new users that are creating an account on a server that is already using v6.0 or later? Does the notice pop-up immediately once they get in the product and interrupt the onboarding flow?